lib/action.rb

```diff
@@ -65,6 +57,7 @@ def fetch_version(ref:)
   def fetch_version_safe(ref:)
     fetch_version(ref: ref)
```

Comment:
erm, this is super minor, but I think I would set `@failed_description = "Update: #{file_path}"` on line 59 rather than in `failed_status_description` - and then rename `failed_status_description` to be something like `truncate_message`. Keeps the division of responsibilities a little cleaner.

Reply:
It is more complicated than I think as some tests are failing. I will send another PR for that.